test/Microsoft.DocAsCode.Common.Tests/RelativePathTest.cs

```diff
@@ -415,5 +415,28 @@ public void TestUrlDecode(string path, string expected)
         {
             Assert.Equal(expected, ((RelativePath)path).UrlDecode());
         }
+
+        [Theory]
+        [InlineData("a/b/c", "a/b/", true)]
+        [InlineData("~/a/b/c", "~/a/b/", true)]
+        [InlineData("a/b/c", "~/a/b/", false)]
+        [InlineData("~/a/b/c", "a/b/", false)]
+        [InlineData("a/b", "a/b", false)]
+        [InlineData("a/b/", "a/b", false)]
+        [InlineData("a/b", "a/b/", false)]
+        [InlineData("a/b/", "a/b/", true)]
+        [InlineData("a/b/c", "a/b/c", false)]
+        [InlineData("a/b/c", "a/b/c/d", false)]
+        [InlineData("a/b/c", "a/b/d", false)]
+        [InlineData("a/../b/c", "b/", true)]
+        [InlineData("../a/b", "../a", false)]
+        [InlineData("../a/b", "../", false)]
+        [InlineData("../a/b", "../../a", false)]
+        [InlineData("../../", "../", false)]
+        [InlineData("../", "../../", false)]
```

Comment:
what about "~/../"?

Reply:
It starts with `../`, which always return false.